### PR TITLE
Move Stable Farms from separate tab to main page.

### PR DESCRIPTION
**Pull request by memacius32**
- Move Stable Farms from separate tab to main page.
- Add Stable farms to main pools search and "Show only my pools"

Screen Shot 2022-05-17 at 11 48 56
Screen Shot 2022-05-17 at 11 49 08

### CODE REVIEW

File: src/pages/EarnTri/index.tsx [hunk 1]
```diff
@@ -1,30 +1,15 @@
...
 
-export default function Earn({
-  match: {
-    params: { farmType = FarmType.NORMAL }
-  }
-}: RouteComponentProps<{ farmType?: FarmType }>) {
+export default function Earn() {
   return (
     <PageWrapper gap="lg" justify="center">
       <MemoizedFarmBanner />
```
Comment (specialist0x) [inline src/pages/EarnTri/index.tsx:11]: Since you're changing this page -- this probably doesn't need to be memoized :)

**Comment by specialist0x** [comment]
Very nice!  Can you check that search and ordering works, too?

**Comment by specialist0x** [comment]
So that the `main` branch doesn't include the `wip` diffs, can you `Squash and merge` this PR?

**Comment by specialist0x** [comment]
You can probably delete `FarmTabs` too

**Review by specialist0x — APPROVED** [review]

File: src/pages/EarnTri/index.tsx (hunk 1 above)
Comment (memacius32) [reply src/pages/EarnTri/index.tsx:11]: you are right sir👏

**Comment by memacius32** [comment]
> Very nice! Can you check that search and ordering works, too?

works

**Pull request merged**